x-pack/plugins/snapshot_restore/public/app/sections/home/repository_list/repository_table/repository_table.tsx

```diff
@@ -53,7 +55,16 @@ const RepositoryTableUi: React.FunctionComponent<Props> = ({
       truncateText: true,
       sortable: true,
       render: (name: Repository['name'], repository: Repository) => {
-        return <EuiLink onClick={() => openRepositoryDetails(name)}>{name}</EuiLink>;
+        return (
+          <EuiLink
+            onClick={() => {
```

Comment:
I just realized we probably want to use href here, since we're changing the URL, right? Then users could open the link in a new window if they wanted to.